CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Watermelon's extension development
 
-For now, we discuss on [Slack](https://join.slack.com/t/watermelonusers/shared_invite/zt-18vhxghk6-7YSg30oWqQSZqdOzlkR~Rw). 
+For now, we discuss on [Slack](https://join.slack.com/t/watermelonusers/shared_invite/zt-18vhxghk6-7YSg30oWqQSZqdOzlkR~Rw).
```

Comment:
Discord [https://discord.gg/H4AE6b9442](https://t.co/fMIlnb9egq)

Reply:
Interesting, why wasn't this changed.
Seems most contributors are on Slack now.

Reply:
We're now putting all our efforts on Discord

Reply:
Why wasn't this changed before?

Reply:
Because I missed it.